docs/components/motor/_index.md

```diff
@@ -68,7 +68,7 @@ await motor2.go_for(1000, 200)
 {{% /tab %}}
 {{% tab name="Go" %}}
 
-Example code should be placed after the `robot, err := client.New(...)` function in `main()`.
+Place the example code after the `robot, err := client.New(...)` function in `main()`.

```

Comment:
```suggestion
Place the example code in the `main()` function after `robot, err := client.New(...)`.
```
(Opt) Ref. to method call as function could be confusing/make people look for function

Reply:
Great point. I dunno why it says this all over the place. Will change :)